syntax="proto3";

package gravity_pb;

import "Dir.proto";

message FloorButtonEntity {
	float posX = 1;
	float posY = 2;
	float posZ = 3;
	Dir dir = 4;
	fixed32 targetName = 5;
	fixed32 sourceIndex = 6;
}
